src/indra_cogex/apps/curator/explorer_blueprint.py

```diff
@@ -362,128 +405,148 @@ def conflicts():
+            The deubiquitinase explorer identifies INDRA statements using INDRA
             CoGEx whose subjects are human deubiquitinase genes and whose
             statements are "deubiquinates".
             {_database_text("Pathway Commons")}
             {EVIDENCE_TEXT}
         """,
+        func_kwargs={'include_db_evidence': include_db_evidence},
+        is_proteocentric=True,
     )
 
 
-@curator_blueprint.route("/mirna", methods=["GET"])
+@explorer_blueprint.route("/mirna", methods=["GET"])
 @jwt_required(optional=True)
 def mirna():
-    """Curate miRNAs."""
+    """Explore miRNAs."""
+    include_db_evidence = request.args.get('include_db_evidence', 'true').lower() == 'true'
     return _render_func(
         get_mirna_statements,
-        title="miRNA Curator",
+        title="miRNA Explorer",
         description=f"""\
-            The miRNA curator identifies INDRA statements using INDRA
+            The miRNA explorer identifies INDRA statements using INDRA
             CoGEx whose subjects are micro-RNAs and whose
             statements are "increases amount" or "decreases amount".
             {_database_text("miRTarBase")}
             {EVIDENCE_TEXT}
         """,
+        func_kwargs={'include_db_evidence': include_db_evidence},
+        is_proteocentric=True,
     )
 
 
-@curator_blueprint.route("/disprot", methods=["GET"])
-@curator_blueprint.route("/disprot/<object_prefix>", methods=["GET"])
+@explorer_blueprint.route("/disprot", methods=["GET"])
+@explorer_blueprint.route("/disprot/<object_prefix>", methods=["GET"])
 @jwt_required(optional=True)
 def disprot(object_prefix: Optional[str] = None):
-    """Curate intrensically disordered proteins."""
+    """Explore intrensically disordered proteins."""
+    include_db_evidence = request.args.get('include_db_evidence', 'true').lower() == 'true'
     assert object_prefix in {None, "hgnc", "go", "chebi"}
     return _render_func(
         get_disprot_statements,
-        func_kwargs=dict(object_prefix=object_prefix),
-        title="DisProt Curator",
+        title="DisProt Explorer",
         description=f"""\
-            The DisProt curator identifies INDRA statements using INDRA
+            The DisProt explorer identifies INDRA statements using INDRA
             CoGEx whose subjects are intrensically disordered proteins.
             {EVIDENCE_TEXT}
         """,
+        func_kwargs=dict(object_prefix=object_prefix, include_db_evidence=include_db_evidence),
+        is_proteocentric=True,
     )
 
 
-@curator_blueprint.route("/modulator/", methods=["GET"])
+@explorer_blueprint.route("/modulator/", methods=["GET"])
 @jwt_required(optional=True)
 def modulator():
     """Get small molecule modulators for the given protein."""
     raise NotImplementedError
 
 
-@curator_blueprint.route("/entity/<prefix>:<identifier>", methods=["GET"])
+@explorer_blueprint.route("/entity/<prefix>:<identifier>", methods=["GET"])
```

Comment:
This should resolve the 404 for when searching publications with DOI (because a DOI typically contains a `/`):
```suggestion
@explorer_blueprint.route("/entity/<prefix>:<path:identifier>", methods=["GET"])
```